cmd/influx/main.go

```diff
@@ -96,7 +96,8 @@ func newApp() cli.App {
 
 // This creates a new slice and replaces `-t "-FOO-TOKEN"` with `-t=-FOO-TOKEN`
 // This is necessary to do because the command line arg:
-//  `-t "-FOO-TOKEN"`` will be parsed as two separate flags instead of a flag and token value.
+//
+//	`-t "-FOO-TOKEN"`` will be parsed as two separate flags instead of a flag and token value.
```

Comment:
I think this comment was more clear before

Reply:
Seems like `gofmt` wants to force it on a newline if it has the leading tab. I removed that to remove the empty line.